ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vlan-interfaces.j2

```diff
@@ -36,7 +36,7 @@
-| {{ vlan_interface }} | {% if vlan_interfaces[vlan_interface].vrf is defined and vlan_interfaces[vlan_interface].vrf is not none %} {{ vlan_interfaces[vlan_interface].vrf }} {% else %} default {% endif %} | {% if vlan_interfaces[vlan_interface].ip_address is defined and vlan_interfaces[vlan_interface].ip_address is not none %} {{ vlan_interfaces[vlan_interface].ip_address }} {% else %} - {% endif %} | {% if vlan_interfaces[vlan_interface].ip_address_virtual is defined and vlan_interfaces[vlan_interface].ip_address_virtual is not none %} {{ vlan_interfaces[vlan_interface].ip_address_virtual }} {% else %} - {% endif %} | {% if vlan_interfaces[vlan_interface].ip_virtual_router_address is defined and vlan_interfaces[vlan_interface].ip_virtual_router_address is not none %} {{ vlan_interfaces[vlan_interface].ip_virtual_router_address }} {% else %} - {% endif %} | {% if vlan_interfaces[vlan_interface].vrrp.ipv4 is defined and vlan_interfaces[vlan_interface].vrrp.ipv4 is not none %} {{ vlan_interfaces[vlan_interface].vrrp.ipv4 }} {% else %} - {% endif %} | {% if vlan_interfaces[vlan_interface].access_group_in is defined and vlan_interfaces[vlan_interface].access_group_in is not none %} {{ vlan_interfaces[vlan_interface].access_group_in }} {% else %} - {% endif %} | {% if vlan_interfaces[vlan_interface].access_group_out is defined and vlan_interfaces[vlan_interface].access_group_out is not none %} {{ vlan_interfaces[vlan_interface].access_group_out }} {% else %} - {% endif %} |
+| {{ vlan_interface }} | {% if vlan_interfaces[vlan_interface].vrf is defined and vlan_interfaces[vlan_interface].vrf is not none %} {{ vlan_interfaces[vlan_interface].vrf }} {% else %} default {% endif %} | {% if vlan_interfaces[vlan_interface].ip_address is defined and vlan_interfaces[vlan_interface].ip_address is not none %} {{ vlan_interfaces[vlan_interface].ip_address }} {% else %} - {% endif %} | {% if vlan_interfaces[vlan_interface].ip_address_virtual is defined and vlan_interfaces[vlan_interface].ip_address_virtual is not none %} {{ vlan_interfaces[vlan_interface].ip_address_virtual }} {% else %} - {% endif %} | {% if vlan_interfaces[vlan_interface].ip_virtual_router_addresses is arista.avd.defined %} {{ vlan_interfaces[vlan_interface].ip_virtual_router_addresses | join(", ") }} {% else %} - {% endif %} | {% if vlan_interfaces[vlan_interface].vrrp.ipv4 is defined and vlan_interfaces[vlan_interface].vrrp.ipv4 is not none %} {{ vlan_interfaces[vlan_interface].vrrp.ipv4 }} {% else %} - {% endif %} | {% if vlan_interfaces[vlan_interface].access_group_in is defined and vlan_interfaces[vlan_interface].access_group_in is not none %} {{ vlan_interfaces[vlan_interface].access_group_in }} {% else %} - {% endif %} | {% if vlan_interfaces[vlan_interface].access_group_out is defined and vlan_interfaces[vlan_interface].access_group_out is not none %} {{ vlan_interfaces[vlan_interface].access_group_out }} {% else %} - {% endif %} |
```

Comment:
We try to avoid inline jinja, since it is hard to read. Please take a look at some of the other documentation templates, where we calculate all values first and then print out the line.
I understand you are only changing an existing template, but we should try to improve the templates when we touch them. Let me know if you need a hand on this.